src/api/app/components/add_review_dropdown_component.html.haml

```diff
@@ -0,0 +1,29 @@
+- if render?
```

Comment:
With your current rules this will also render if I *can* possibly review but there isn't a review for me...

![Screenshot from 2022-08-09 17-10-07](https://user-images.githubusercontent.com/514785/183687428-58945ef8-48fe-4564-a9d3-8361757e2a04.png)

Reply:
Agree, we renamed it to just `Review`